packages/core/src/services/ai/index.ts

```diff
@@ -161,3 +161,5 @@ export function validateConfig(config: Record<string, unknown>): Config {
 
   return configSchema.parse(config)
 }
+
+export { estimateCost } from './costEstimate'
```

Comment:
estimateCost -> costEstimate. Why not same func and filename? : )

Reply:
you got a point there sir